### ISSUE TITLE
Add support for cloud_firestore 0.16.0

### ISSUE THREAD

**Issue by VOIDCRUSHER** [atn832/cloud_firestore_mocks#138]
Getting the following error message after upgrading to firestore 0.16.0 from the previously supported 0.14.0:

```
Because cloud_firestore_mocks 0.5.2+1 depends on cloud_firestore ^0.14.0+2 and {PROJECTNAME} depends on cloud_firestore ^0.16.0, cloud_firestore_mocks 0.5.2+1 is forbidden.
So, because {PROJECTNAME} depends on cloud_firestore_mocks 0.5.2+1, version solving failed.
pub get failed (1; So, because {PROJECTNAME} depends on cloud_firestore_mocks 0.5.2+1, version solving failed.)

```

From the firebase changelogs, It appears that the main breaking changes in the new versions is the removal of all previously deprecated functions and a min IOS 10.12 requirement.

**Comment by atn832** [comment]
Thanks for reporting. You're right, it was a quick fix. I just published 0.6.0, which will let you use firestore 0.16.0. https://pub.dev/packages/cloud_firestore_mocks/changelog#060

**Comment by VOIDCRUSHER** [comment]
That was super quick. Thanks for maintaining this package!
